docs/guides/distribution/sign-linux.md

```diff
@@ -26,7 +26,7 @@ You can embed a signature in the AppImage by setting the following environment v
 
 - **SIGN**: set to `1` to sign the AppImage.
 - **SIGN_KEY**: optional variable to use a specific GPG Key ID for signing.
-- **APPIMAGETOOL_SIGN_PASSPHRASE**: the signing key password. If unset, gpg shows a dialog so you can input it, so you must set this when running in automated tasks.
+- **APPIMAGETOOL_SIGN_PASSPHRASE**: the signing key password. If unset, gpg shows a dialog so you can input it. So you must set this when running automated tasks.
```

Comment:
why `. So` instead of `, so`?

Reply:
At first glance, it seemed like it was most appropriate to divide the sentence into two parts, as there is a repeating `so`.
That being said, I'm unsure if doing so causes this description to lose meaning in the context of the documented item.

Reply:
Maybe remove `So` from the second sentence.